mimic/model/nova_objects.py

```diff
@@ -418,27 +417,88 @@ def hooked(collection, http, json, absolutize_url):
     return hooked
 
 
-@server_creation.declare_behavior_creator("fail")
-def create_fail_behavior(parameters):
+def _get_failure_behavior(parameters, create=False):
     """
-    Create a failing behavior for server creation.
+    Helper function to produce a failure to create function.  Either creating
```

Comment:
One minor nitpick: I try to avoid ever saying the type of the thing that I'm documenting in the documentation.  Otherwise eventually every class would start with "This is a class." and every module would start with "This is a module."

Also, "Either creating the server or not." is a sentence fragment.

This is a little tricky to describe but try to stick to its purpose: "Build a response body indicating a failure in Nova…"; something like that.

Also, given the plethora of types might be talking about when one describes a "response" or even a "response body", an `:rtype:` and `:return:` annotation would be useful here.